java/connector-node/risingwave-connector-service/src/main/java/com/risingwave/sourcenode/postgres/PostgresSourceConfig.java

```diff
@@ -107,4 +106,9 @@ public SourceTypeE getSourceType() {
     public Properties getProperties() {
         return props;
     }
+
+    private static String generateReplicaSlot() {
```

Comment:
Although MZ can generate the slot name internally, I think it is necessary to store the slot name somewhere. Then upon a recovery, we can reuse the previously created slot instead of creating a new one. At before, the slot name is stored in the `properties` of Source in the catalog. So we may need to generate the slot name on Meta.

Reply:
Got it. Let me move this part to Meta.